features/camel/src/main/feature/feature.xml

```diff
@@ -235,7 +235,7 @@
     </feature>
 
   <feature name="camel-quartz2" version="${camel.version}" start-level="50">
-        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}</bundle>
+        <bundle dependency="true">wrap:mvn:com.mchange/c3p0/${c3p0-bundle-version}</bundle>
```

Comment:
❓ I'll defer to @emmberk here, but I think we typically prefer comments on why these types of changes are needed when we stray from what comes standard.

Reply:
Thanks for the ping.

❓Why is this newer version needed? The ticket says "better XML processing". What does that mean for DDF? Is this change required for future DDF tickets? Why is this improvement only needed for camel? Is there a security concern with `mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.c3p0/${c3p0-bundle-version}`? Can this be accomplished with a `bundleReplacement` in `org.apache.karaf.features.xml`, instead?

❓Why is `wrap:` needed?

Yes, we should document any changes between `<repository>mvn:org.apache.camel.karaf/apache-camel/2.22.0/xml/features</repository>` and what's defined in this custom repo for DDF here: https://github.com/codice/ddf/blob/19eb4a29c6b9990ccd01828ee0c91b853d5fec67/features/camel/src/main/feature/feature.xml#L28-L48. The note should say why the change is needed to that judgement can be used for future changes. For example, if camel is upgraded to a new version, should this change be applied to the new custom `apache-camel` features repo?

Reply:
>Why is this only needed for Camel?

Because Camel's `quartz2` component is driving the need for this particular flavor of the dependency (https://github.com/apache/camel/blob/camel-2.23.1/parent/pom.xml#L103-L104); they have not upgraded yet and service mix has not published a newer version either. 

>Why is `wrap` needed?

See above; lack of unpublished dependencies. Also, because the artifact is not a bundle: https://search.maven.org/artifact/com.mchange/c3p0/0.9.5.3/jar

>Is `c3p0-bundle-version` defined anywhere else?

No. Its only use is here, in the features file. Could look into removing that property. Will also add a comment with more details. When Camel itself is upgraded, there is a possibility the source of the artifact can return to service mix.